### PR TITLE
update Polish bank list

### PR DESCRIPTION
**Pull request by IonBazan**
The Polish bank institution list has changed over these 5 years. I've downloaded it from https://ewib.nbp.pl/, ordered by bank code and removed duplicates.

### CODE REVIEW

**Comment by IonBazan** [comment]
Rebased onto `master` after merging #1469 for tests to pass.

File: src/Faker/Provider/pl_PL/Payment.php [hunk 1]
```diff
@@ -5,81 +5,76 @@
 class Payment extends \Faker\Provider\Payment
 {
     /**
-     * @var array list of Polish banks, source: http://www.nbp.pl/banki_w_polsce/ewidencja/ewidencja.html
+     * @var array list of Polish banks, source: https://ewib.nbp.pl/
```
Comment (ppelgrims) [inline src/Faker/Provider/pl_PL/Payment.php]: Does this dataset comply with the MIT license of this package? Or is it copyrighted?

Comment (Saibamen) [reply src/Faker/Provider/pl_PL/Payment.php]: I think this is free for OS projects. I already send an e-mail to NBP with your question

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**